checks/shell_download_validate.go

```diff
@@ -35,8 +35,11 @@ var ErrParsingDockerfile = errors.New("file cannot be parsed")
 var ErrParsingShellCommand = errors.New("shell command cannot be parsed")
 
 // List of interpreters.
+var pythonInterpreters = []string{"python", "python3", "python2.7"}
+
 var interpreters = []string{
```

Comment:
Change to:

```
var interpreters = append([]string{"sh", <>}, pythonInterpreters...)
```

Reply:
thank you! Was looking for some better way!